packages/itwinui-react/src/core/Header/HeaderBasicButton.tsx

```diff
@@ -33,15 +33,9 @@ export const HeaderBasicButton = React.forwardRef((props, ref) => {
       style={style}
       {...rest}
     >
-      {startIcon &&
-        React.cloneElement(startIcon, {
-          className: startIcon.props.className,
-        })}
+      {startIcon}
```

Comment:
What is the difference? i.e why was `cloneElement` used in the first place? Does it mean that we could achieve the same thing but with CSS adjustments?

Reply:
if you're asking only about `HeaderBasicButton`, i think in this case it did nothing because we are simply assigning the classname that is already there